x-pack/plugins/task_manager/server/task_running/task_runner.ts

```diff
@@ -717,3 +715,15 @@ export function asRan(task: InstanceOf<TaskRunningStage.RAN, RanTask>): RanTask
     task,
   };
 }
+
+export function calculateDelay(attempts: number) {
```

Comment:
I guess we never set a global max on the number of retries task manager will attempt if there's no custom retry logic? Wondering if we should... cc @mikecote

Reply:
Ok...I see the logic now...sorry about that!